design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -111,12 +114,19 @@ input CloseNegotiableQuotesInput {
 }
 
 type CloseNegotiableQuotesOutput {
-    closed_quotes: [NegotiableQuote!] @doc(description: "Quotes that were just closed")
+    status: Boolean!
+    closed_quotes(
```

Comment:
[NegotiableQuote!] - would need NegotiableQuotesOutput { items: [NegotiableQuote!]}

while NegotiableQuotesOutput currently has  items: [NegotiableQuote]!

Should we add a specific type for both? or modify NegotiableQuotesOutput to have
 items: [NegotiableQuote] - basically satisfying both cases but leaving schema less specific

Reply:
For reference regarding nullability of array members: https://www.apollographql.com/blog/using-nullability-in-graphql-2254f84c4ed7

```
drinkSizes: [String!]

This means that the list itself can be null, but it can’t have any null members. For example, in JSON:

drinkSizes: null // valid
drinkSizes: [] // valid
drinkSizes: ["small", "large"] // valid
drinkSizes: ["small", null, "large"] // error
```

Reply:
so it seems that we should go for `items: [NegotiableQuote!]!`

Reply:
Adding more specificity is risky as it tends to be the least BIC. I would suggest items: [NegotiableQuote] for now so it can be consistent with the rest of the schema. We can always add more specificity later.

Reply:
@cpartica since we are not tackling status now, can you remove the status fields from this PR. Rest looks good.

Reply:
done